### ISSUE TITLE
Setup static code checks

### ISSUE THREAD

**Issue by maciaszczykm** [kubermatic/dashboard#809]
Currently in the CI we are checking `make lint` results but we should also check files formatting to keep it always the same across the repository.

Here is a list of things that should be done:
- [x] lint check
- [ ] lint staged
- [x] format check
- [x] command running all static checks [in parallel](http://docs.drone.io/pipelines/)
- [ ] option to add formatter to IDE's save hooks*
- [ ] pre-commit hook running all static checks**
- [x] formatting all files
- [x] update of CI pipeline

&ast; It depends on IDE's support of formatter that we use. For example [WebStorm supports TSLint out of the box](https://intellij-support.jetbrains.com/hc/en-us/community/posts/360000047844-Angular-and-Tslint-for-code-reformat), [VS Code has a plugin for it](https://marketplace.visualstudio.com/items?itemName=eg2.tslint) etc.
&ast;&ast; Not required, but in my opinion it is nice to have.

We should also consider impact that formatting will have on work on multiple branches, i.e. moving commits between them.
